### PR TITLE
Homework1

### PR DESCRIPTION
**Pull request by Gunita-Vike-Daudzvarde**
This is my first homework. Be gentle.

### CODE REVIEW

File: src/lv/lu/homework1/MyFirstHomework.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package lv.lu.homework1;
+
```
Comment (Gunita-Vike-Daudzvarde) [inline src/lv/lu/homework1/MyFirstHomework.java:2]: //good

**Review by GuntarsR — COMMENTED** [review]
Looks good!

**Review by GuntarsR — APPROVED** [review]
OK!

**Review by GuntarsR — APPROVED** [review]
OK!

**Pull request merged**